src/Calendar_Holiday.php

```diff
@@ -106,8 +106,7 @@ public static function showForCalendar(Calendar $calendar)
 
         if ($canedit) {
             TemplateRenderer::getInstance()->display('pages/setup/calendar_holiday.html.twig', [
-                'item' => new self(),
-                'calendars_id' => $ID,
```

Comment:
I don't mind it personally, but technically changing the template parameters is a breaking change.

It is unlikely that a plugin re-use this template so we can let it go but we should be careful about this kind of changes when we are in a RC state.